src/services/ldap/strategies/index.js

```diff
@@ -1,11 +1,9 @@
 const UniventionLDAPStrategy = require('./univention');
-const iServLDAPStrategy = require('./iserv');
 const GeneralLDAPStrategy = require('./general');
 const IservIdmLDAPStrategy = require('./iserv-idm');
 
 const strategies = {
 	univention: UniventionLDAPStrategy,
-	iserv: iServLDAPStrategy,
```

Comment:
why removing iserv strategy? Surely is not part of this ticket

Reply:
its part of https://ticketsystem.dbildungscloud.de/browse/N21-871
I wrote it in their ticket, but not in the PR, will note it here as well